datasets/modeled/transport/dataset.json5

```diff
@@ -1,59 +1,94 @@
 {
-  dataset_id: "tempo_conus_2022",
+  dataset_id: "decarb_2023_transport",
   dataset_type: "modeled",
-  data_source: "tempo",
-  sector_description: "transportation",
-  description: "CONUS 2022 TEMPO county-level results for PEV charging.",
+  data_source: "tempo evix polaris",
+  sector_description: "electric load associated with light, medium, and heavy-duty road transportation",
+  description: "DECARB 6 results from TEMPO, EVI-X, and POLARIS (Nov 2023) with profiles using 2018 as the weather year",
   origin_creator: "Arthur Yip",
   origin_organization: "NREL",
   origin_contributors: [
     "Arthur Yip",
+    "Brennan Borlaug"
   ],
-  origin_project: "",
-  origin_date: "",
-  origin_version: "",
-  source: "",
+  origin_project: "DECARB 6",
+  origin_date: "2023-11-30",
+  origin_version: "0.1.0",
+  source: "TEMPO, EVI-X, and POLARIS simulations for DECARB 6",
   data_classification: "low",
   tags: [
-    "transportation",
-    "tempo",
+    "TEMPO",
+    "EVIX",
+    "POLARIS",
+    "EV",
+    "decarb",
   ],
+  data_schema_type: "standard",
   trivial_dimensions: [
-    "sector",
     "weather_year",
+    "sector"
   ],
   use_project_geography_time_zone: true,
   data_schema: {
     table_format: {
-      format_type: "pivoted",
-      pivoted_dimension_type: "subsector",
+      format_type: "unpivoted",
     },
     data_schema_type: "one_table",
   },
   dimensions: [
-    {
-      "class": "County",
+    { 
+     "class": "County",
       type: "geography",
-      name: "ACS County 2018",
+      name: "US counties 2020 L48",
       display_name: "County",
       file: "dimensions/counties.csv",
-      description: "American Community Survey US counties, 2018.",
+      module: "dsgrid.dimension.standard",
+      description: "US counties for L48, Census 2020 vintage, no territories",
```

Comment:
If the list is exactly the same as the project, I'd directly reuse all that information.

Reply:
The data includes some (not all) counties from Alaska and Hawaii, so this name and description should be changed.